### PR TITLE
2018-02-07 opcodes meeting summary

### PR DESCRIPTION
**Pull request by Danconnolly**

### CODE REVIEW

File: wg-opcodes/summaries/20180207 - Meeting Summary.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Chair:
+ * JVP
+ 
+Developer Participants:
+ * Shammah Chancellor of Bitcoin ABC
+ * Dan Connolly of BitcoinJ Cash
+ * Amaury Sechét of Bitcoin ABC
+ * Steve Shadders of nChain
+ * Andrew Stone of Bitcoin Unlimited
+ * *more*
```
Comment (sickpig) [inline wg-opcodes/summaries/20180207 - Meeting Summary.md]: Andrea Suisani of Bitcoin Unlimited

**Review by sickpig — APPROVED** [review]

File: wg-opcodes/summaries/20180207 - Meeting Summary.md (hunk 1 above)
Comment (schancel) [reply wg-opcodes/summaries/20180207 - Meeting Summary.md]: Jason B Cox
Joshua Yabut

**Comment by schancel** [comment]
Looks good.

**Comment by Danconnolly** [comment]
I added the attendees that were suggested.

**Pull request merged**